test/integration/iso_test.go

```diff
@@ -0,0 +1,65 @@
+// +build iso
+
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"k8s.io/minikube/test/integration/util"
+)
+
+func TestISO(t *testing.T) {
+
+	minikubeRunner := util.MinikubeRunner{
+		Args:       *args,
+		BinaryPath: *binaryPath,
+		T:          t}
+
+	minikubeRunner.RunCommand("delete", true)
+	minikubeRunner.Start()
```

Comment:
The idea of this test is to test new ISOs, so we want to make sure whatever ISO is passed into m.Args is picked up.